make/config/lib.sh

```diff
@@ -36,7 +36,18 @@ warn=
 wait=
 greencheck=
 redcross=
-if ! printenv NO_COLOR >/dev/null; then
+
+should_color() {
+	if [[ "${CI:-}" == "true" ]]; then
+		return 1
+	elif [[ "${NO_COLOR:-}" ]]; then
+		return 1
+	fi
+
+	return 0
+}
```

Comment:
I think the CI system should have set NO_COLOR already right? https://www.gnu.org/software/gettext/libtextstyle/manual/html_node/The-NO_005fCOLOR-variable.html

Reply:
`NO_COLOR` isn't listed in the [list](https://docs.prow.k8s.io/docs/jobs/#job-environment-variables)!

Reply:
Maybe other tools also support NO_COLOR; would it make sense to set this on a higher level (eg. globally in the makefile)?

Reply:
> would it make sense to set this on a higher level

Sure, but not in this PR!